### PR TITLE
Fixing several small site bugs / requests

### PR DESCRIPTION
**Pull request by andrewconner**
Elaboration in comments.

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/libraries/deleteLibraryConfirmModal.tpl.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div kf-modal class="kf-delete-library-confirm-modal">
   <div kf-basic-modal-content with-cancel="true" title="Are you sure?" with-warning action-text="Delete Library" action="deleteLibrary()">
-    Deleting your library will remove all your keeps in the library!
+    <p>Deleting your library will remove all your keeps and followers in the library.</p>
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/libraries/deleteLibraryConfirmModal.tpl.html]: https://app.asana.com/0/15523651812135/18182167176068

File: kifi-backend/modules/common/app/com/keepit/model/Library.scala [hunk 2]
```diff
@@ -68,7 +68,7 @@ object Library extends ModelWithPublicIdCompanion[Library] {
     (__ \ 'lastKept).formatNullable[DateTime]
   )(Library.apply, unlift(Library.unapply))
 
-  val maxNameLength = 50
+  val maxNameLength = 200
```
Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/model/Library.scala:71]: https://app.asana.com/0/15523651812135/18173507722383
Need frontend validation!

File: kifi-backend/modules/shoebox/angular/src/common/services/twitter.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
+'user strict';
+
+angular.module('kifi')
+
+.provider('$twitter', function $twttrProvider() {
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/common/services/twitter.js:5]: https://app.asana.com/0/15523651812135/17379239993725

File: kifi-backend/modules/shoebox/angular/src/layout/nav/nav.tpl.html [hunk 4]
```diff
@@ -22,13 +22,13 @@
...
 
-  <div ng-show="mainLib.numKeeps >= 0">
+  <div>
     <ul class="kf-nav-lib-system">
       <li class="kf-nav-lib-item" ng-class="{ active: isActive('mainLib.url') }">
         <a class="kf-nav-link" ng-href="{{mainLib.url}}">
           <div class="library-icon svg-main-library-icon"></div>
           <div class="kf-nav-lib-info">
-            <div class="kf-nav-lib-name">{{mainLib.name}}</div>
+            <div class="kf-nav-lib-name">My Main Library</div>
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/layout/nav/nav.tpl.html:31]: Cosmetic only, need to rename all system libraries.

https://app.asana.com/0/15523651812135/18287706756702

File: kifi-backend/modules/shoebox/angular/src/friends/rightColConnectView.tpl.html [hunk 5]
```diff
@@ -1,5 +0,0 @@
-<div class="kf-rightcol-friends-connect" ng-show="connectSocial">
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/friends/rightColConnectView.tpl.html:1]: https://app.asana.com/0/15523651812135/18287706756704

File: kifi-backend/modules/shoebox/angular/src/friends/rightColFriendsView.tpl.html [hunk 6]
```diff
@@ -1,6 +1,5 @@
 <div>
+  <div kf-people-you-may-know-view></div>
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/friends/rightColFriendsView.tpl.html:2]: https://app.asana.com/0/15523651812135/18287706756708

**Comment by andrewconner** [comment]
@yipingliao

File: kifi-backend/modules/shoebox/angular/src/libraries/deleteLibraryConfirmModal.tpl.html (hunk 1 above)
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/libraries/deleteLibraryConfirmModal.tpl.html]: “remove all your keeps” isn’t clear about whether the keeps will still exist somewhere else. “delete” or “destroy” would be clearer. Also, “followers in the library” sounds like a typo. Maybe:
“Deleting your library will delete all keeps in the library and remove all of its followers.”

File: kifi-backend/modules/shoebox/angular/src/tagManage/deleteTagModal.tpl.html [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <div kf-modal class="kf-delete-tag-confirm-modal">
   <div kf-basic-modal-content with-cancel="true" title="Are you sure?" with-warning action-text="Delete Tag" action="deleteTag()">
-    Deleting this tag will remove all keeps with this tag!
+    Are you sure you want to delete this tag?
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/tagManage/deleteTagModal.tpl.html]: I think it’s important to explain clearly what the action does, as we do for the delete library action. Maybe:
“Deleting this tag will remove it from N keeps.”

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/tagManage/deleteTagModal.tpl.html]: Thanks, fixed.

**Comment by andrewconner** [comment]
Thanks for the feedback @2is10. Modified.

**Pull request merged**